scraper/data/detail_yvonne.csv

```diff
@@ -0,0 +1,31 @@
+Folder Name,Common Name,Common Name DE,Subfamily,To check,Is Tree,Mature Size Height,Mature Size Width,Nutrition Demand,Binomial name,Genus,Family,Edible uses,Medicinal uses,Material uses &amp; Functions,Botanic,Propagation,Cultivation,Environment,Material uses,Functions,Provides forage for,Provides shelter for,Hardiness Zone,Heat Zone,Water,Sun,Shade,Soil PH,Soil Texture,Soil Water Retention,Environmental Tolerances,Native Climate Zones,Adapted Climate Zones,Native Geographical Range,Native Environment,Ecosystem Niche,Root Zone Tendancy,Deciduous or Evergreen,Herbaceous or Woody,Life Cycle,Growth Rate,Fertility,Pollinators,Flower Colour,Flower Type,Has drought tolerance,Tolerates wind,Plant References

```

Comment:
Please add a README.md how this merging works (what is Yvonne allowed to change etc.)